tests/i18n/tests.py

```diff
@@ -1415,6 +1416,29 @@ def test_i18n_patterns_returns_list(self):
         with override_settings(USE_I18N=True):
             self.assertIsInstance(i18n_patterns([]), list)
 
+    @override_settings(
+        LANGUAGE_CODE='pt',
+        LOCALE_PATHS=[
+            os.path.join(here, 'locale'),
+            (os.path.join(here, 'locale'), 'custom_domain')
```

Comment:
Include a trailing comma in cases like this so that if more items are added later, this line doesn't have to be modified again.